server/app/services/question/LocalizedQuestionOption.java

```diff
@@ -22,6 +22,9 @@ public static LocalizedQuestionOption create(
   /** The order of the option. */
   public abstract long order();
 
+  /** The api-facing admin name */
+  public abstract String adminName();
```

Comment:
**suggestion (blocking):** A little more explanation here could be helpful I think since we use the admin name for identifying options in predicates as well.
```suggestion
  /** The immutable identifier for this options used to reference it in the API and predicates. */
  public abstract String adminName();
```

Reply:
Done.